app/vue3/README.md

```diff
@@ -0,0 +1,41 @@
+# Storybook for Vue 3
+
+Storybook for Vue 3 is a UI development environment for your Vue 3 components.
+With it, you can visualize different states of your UI components and develop them interactively.
+
+![Storybook Screenshot](https://github.com/storybookjs/storybook/blob/master/media/storybook-intro.gif)
+
+Storybook runs outside of your app.
+So you can develop UI components in isolation without worrying about app specific dependencies and requirements.
+
+## Getting Started
+
+```sh
+cd my-vue3-app
+npx -p @storybook/cli sb init
+```
+
+For more information visit: [storybook.js.org](https://storybook.js.org)
+
+---
+
+Storybook also comes with a lot of [addons](https://storybook.js.org/docs/vue3/configure/storybook-addons) and a great API to customize as you wish.
```

Comment:
How does `vue3` section get generated on the docs website?

Reply:
This needs to be added in the frontpage repo. @jonniebigodes  can help with that.